src/js/player.js

```diff
@@ -49,7 +49,12 @@ vjs.Player = vjs.Component.extend({
     this.language_ = options['language'] || vjs.options['language'];
 
     // Update Supported Languages
-    this.languages_ = options['languages'] || vjs.options['languages'];
+    var languages = vjs.obj.copy(options['languages'] || vjs.options['languages']);
+    vjs.options['languages'] = {};
```

Comment:
Ah, ok. There are two levels of options in video.js: page-level and player-level. I believe this code was originally intending to use player-level language settings if provided and fall back to the page-level language settings otherwise.